app/controllers/application_controller.rb

```diff
@@ -111,6 +113,12 @@ def redirect_back_to_referer(fallback_location:)
 
   private
 
+  # Allows us to not specify respond_to formats in json-only controller or action.
+  # Same behavior as when a request format is not defined in a respond_to block.
+  def force_json_format
+    raise ActionController::UnknownFormat unless request.format.json?
+  end
```

Comment:
Is this comment clear?

Avoids this:
```ruby
if @additional_expense.save
    render json: @additional_expense.as_json, status: :created
else
    render json: @additional_expense.errors.as_json, status: :unprocessable_entity
end
```
Needing to be (something like) this:
```ruby
respond_to do |format|
  if @additional_expense.save
    format.json { render json: @additional_expense.as_json, status: :created }
  else
    format.json { render json: @additional_expense.errors.as_json, status: :unprocessable_entity }
  end
end
```

Reply:
That worked, but was just kind of a strange pattern to me -- I felt like this was a more rails-y way to do it/other people could read & understand it better. I would look in the routes file AFTER the controller, anyway.